packages/core-app-api/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Patch Changes
 
+- Reverted the `createApp` TypeScript type to match the one before version `0.1.21`, as it was an accidental breaking change.
```

Comment:
Forgot to change the version number above

Reply:
Yeah ended up running it through the `yarn release` process instead and cleaned up unrelated stuff, hopefully fine now